<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecord">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output selected as MP3</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstSignals">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenLog">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Open log...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeOutput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Change output path...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutputPath">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlayback">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Playback rate (Hz or 1/sec):</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionChange_Path">
   <property name="text">
    <string>Change Path</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>openLog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked()</signal>
   <receiver>actionPlay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRecord</sender>
   <signal>clicked()</signal>
   <receiver>actionRecord</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChangeOutput</sender>
   <signal>clicked()</signal>
   <receiver>actionChange_Path</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenLog</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenLog</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
